sdlib/d/gc/page.d

```diff
@@ -773,6 +775,22 @@ private:
 		}
 	}
 
+	static void zeroSlabNimble(ulong evicted, int sizeClass, Extent* e,
+	                           size_t nimbleIndex) {
+		assert(evicted != 0, "No slabs to clear!");
+		import d.gc.slab;
+		auto slotSize = binInfos[sizeClass].slotSize;
+		void* nimbleBase = e.address + (nimbleIndex * 64 * slotSize);
+
+		while (evicted != 0) {
+			auto index = countTrailingZeros(evicted);
+			void* ptr = nimbleBase + index * slotSize;
+
+			memset(ptr, 0, slotSize);
+			evicted &= (evicted - 1);
+		}
```

Comment:
You probably want to use a technique similar to what is used in block to find range or slots to clean, and clean them all at once, which gives the opportunity to memset to do it in bulk.